src/boilerplate/common/services/generic-api_services.mjs

```diff
@@ -25,8 +26,8 @@ let encryption = {};
 
 export async function service_FUNCTION_NAME (req, res, next){
   try {
-    await web3.connect();
-    await new Promise((resolve) => setTimeout(() => resolve(), 3000));
+    await web3Instance.connect();
```

Comment:
I was puzzled about why we call `.connect()`. Does not the import of the `web3Instance` object cause the constructor to run, calling `this.connect()`, and therefore `web3Instance` is already connected, in which case why is this call needed?